crypto/dilithium/pqcrystals_dilithium_ref_common/sign.c

```diff
@@ -270,28 +344,28 @@ int crypto_sign_verify(ml_dsa_params *params,
   polyveck t1, w1, h;
   keccak_state state;
 
-  if(ctxlen > 255 || siglen != params->bytes)
+  if(siglen != params->bytes) {
     return -1;
-
+  }
+  /* FIPS 204: line 1 */
   unpack_pk(params, rho, &t1, pk);
-  if(unpack_sig(params, c, &z, &h, sig))
+  if(unpack_sig(params, c, &z, &h, sig)) {
     return -1;
-  if(polyvecl_chknorm(params, &z, params->gamma1 - params->beta))
+  }
+  if(polyvecl_chknorm(params, &z, params->gamma1 - params->beta)) {
     return -1;
+  }
 
-  /* Compute CRH(H(rho, t1), msg) */
+  /* FIPS 204: line 7 Compute CRH(H(rho, t1), msg) */
```

Comment:
can you please clarify the discrepancy between the spec and the code. the spec says:

6: 𝑡𝑟 ← H(𝑝𝑘, 64)
7: 𝜇 ← (H(BytesToBits(𝑡𝑟)||𝑀 ′, 64))

Reply:
Great call out, added in [dfd1601](https://github.com/aws/aws-lc/pull/1999/commits/dfd1601d19493fedf2602ae2f3c1a2de52173055)